spec/models/comment_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject { User.new(Name: 'Tom', Image: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'Teacher from Mexico.') }
+
+  before { subject.save }
+
+  before(:each) do
+    @post = Post.create(AuthorId: subject.id, Title: 'dummy post', Text: 'This is a test for the latest_posts method')
```

Comment:
As the previous code reviewer suggested kindly fix your tests. Whenever I try running your tests Im getting the error below.

![test](https://user-images.githubusercontent.com/79506105/215033102-b062d006-5031-4e23-b831-4d10f7a61a64.PNG)